lib/message_handler.rb

```diff
@@ -11,43 +11,62 @@ class MessageHandler < FeatureFlag
   release_for 'Fabien Townsend'
 
   def initialize(args = {})
-    @mark_all_out = args[:mark_all_out]
+    @mark_all_out = args[:mark_all_out] # TODO - this doesnt seem to be used here.
```

Comment:
I think it would be better to create an issue than push a TODO comment on master